### PR TITLE
chore: update changelog for appium 1.22.2

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

For https://github.com/appium/appium/pull/16254

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+===================================
+
+Appium 1.22.2 is a patch release
+
+We're going to move to [Appium 2.x](https://github.com/appium/appium/blob/2.0/docs/en/about-appium/1.x-eol.md).
+Appium 1.x will be receiving minor or patch releases only if XCTest gets breaking updates or major bugs pop up before the EOL.
+
+### iOS(XCUITest)
+
+- Add `safariTabBarPosition` settings api to help `nativeWebTap` capability/setting consider if the device has the tab bar on the top of Safari or on the bottom. Please read `safariTabBarPosition` in [Settings API](https://github.com/appium/appium-xcuitest-driver#settings-api) for more details [appium-xcuitest-driver#1361](https://github.com/appium/appium-xcuitest-driver/pull/1361)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:11]: at the top or at the bottom of Safari window

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+- Do not show the keyboard tutorial in iOS 15 environemnt for simulators [appium-ios-simulator#315](https://github.com/appium/appium-ios-simulator/pull/315)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:12]: typo in environemnt

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+- Fix to pass installation options in [Install App](http://appium.io/docs/en/commands/device/app/install-app/) command [appium-xcuitest-driver#1357](https://github.com/appium/appium-xcuitest-driver/pull/1357)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:13]: should we also mention https://github.com/appium/WebDriverAgent/pull/540 and https://github.com/appium/WebDriverAgent/pull/544 ?

Comment (KazuCocoa) [reply CHANGELOG.md:13]: How should address https://github.com/appium/WebDriverAgent/pull/544 ? I've appended as a performance improvement tho.

Comment (mykola-mokhnach) [reply CHANGELOG.md:13]: yes, performance makes sense.

**Comment by KazuCocoa** [comment]
Updated in 1d578ea.
RC https://www.npmjs.com/package/appium/v/1.22.2-rc.0 also seems good so far.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+
+### iOS(XCUITest)
+
+- Add `safariTabBarPosition` settings api to help `nativeWebTap` capability/setting consider if the device has the tab bar at the top or at the bottom of Safari window. Please read `safariTabBarPosition` in [Settings API](https://github.com/appium/appium-xcuitest-driver#settings-api) for more details [appium-xcuitest-driver#1361](https://github.com/appium/appium-xcuitest-driver/pull/1361)
+- Do not show the keyboard tutorial in iOS 15 environment for simulators [appium-ios-simulator#315](https://github.com/appium/appium-ios-simulator/pull/315)
+- Fix to pass installation options in [Install App](http://appium.io/docs/en/commands/device/app/install-app/) command [appium-xcuitest-driver#1357](https://github.com/appium/appium-xcuitest-driver/pull/1357)
+- Disable the default XCTest's notification checker to avoid [an issue on iOS 15.2](https://github.com/appium/appium/issues/16025) [WebDriverAgent#540](https://github.com/appium/WebDriverAgent/pull/540)
+- Improve generating XML source performance [appium-xcuitest-driver#1351](https://github.com/appium/appium-xcuitest-driver/pull/1351) [WebDriverAgent#544](https://github.com/appium/WebDriverAgent/pull/544)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:15]: Slightly improve the performance of XML source generation

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**